### PR TITLE
chore(segment): remove Segment

### PR DESCRIPTION
**Pull request by rafaelromcar-parabol**
# Description
Removes all references to Segment

### CODE REVIEW

**Comment by rafaelromcar-parabol** [comment]
@tianrunhe you told me one month ago that we could remove segment from the .env.example and I've done that in this PR, but I would like you to validate that Segment isn't used anywhere. I also see some stuff related to Segment on the yarn.lock, but maybe I'm wrong.

Can you do the following?
- Check if there is anything else to remove and, if there is, remove it.
- Merge this PR once you have done that

Thanks

**Comment by dbumblis-parabol** [comment]
LGTM, I will defer to @tianrunhe

**Review by tianrunhe — APPROVED** [review]

**Pull request merged**
